Best practices/Clean code.md

```diff
@@ -59,12 +59,23 @@ The logic contained in a single method should reflect the same level of abstract
 * Methods should not have multiple return params (exception is TryDoSomething pattern which returns bool and resulting object via out param).
 * Avoid using flag arguments. Split the method into several independent methods that can be called from the client without the flag.
 
+### Code order
+
+C# doesn't have any specific requirements for the order of the code inside a class. This is great for us because it gives us the freedom to place code wherever we want, but that doesn't mean that we should just put it anywhere and call it a day.
```

Comment:
`... specific requirements for the code order inside a class` maybe? It reads better to me.

Reply:
I agree, changed!